resources/lang/nl/validation.php

```diff
@@ -1,7 +1,7 @@
 <?php
```

Comment:
Changes in this file will be overwritten by sync with [Laravel-Lang/Lang](https://github.com/Laravel-Lang/lang). Consider contributing to that repo too.

Reply:
Good idea, will do.